sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -1,18 +1,233 @@
 
 Setting up KA Lite
-------------------------
+-------------------
+Once you have successfully installed KA Lite, the installation script should give you a URL (http://127.0.0.1:8008/) to visit so that you can open KA Lite and login for the first time. 
+
+Copy and paste the URL into a web browser. The KA Lite application should show up.  and login to KA Lite using the username and password you created during the installation process. 
+
+* If you have forgotten the username/password combination, simply delete this version of KA Lite (delete the “ka-lite” folder that you downloaded during the installation guide steps for your system and then redo the installation steps in the Install Guide). If it is critical that you are able to login with your credentials, :doc:`../contact` and we can help you manually reset your login information.
+
+Once you’ve logged in, the next step in the setup process is registering your device with the KA Lite Hub.
 
 Registering Your Device with the Hub
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+By registering your device with FLE, you can sync data back with our central data hub. This is useful for many reasons:
+
+    #. a project administrator can manage user accounts and view usage data from afar, without needing to physically visit the offline device.
+    #. usage data syncing back can inform the FLE team of multiple users in a certain geographic region, and we can connect people that might be able to help one another
+    #. it helps FLE and our partners understand where and how the software is being used, so we can keep adding features that support you! 
+
+You will have two options: 
+
+    #. **One click registration**. This is the perfect option for individual users who just want to get KA Lite up and running fast, and don't need online access to data. This allows you to get the registration process over in one-click without worrying about creating a login that you're never going to use. **Caution**: if you choose to one-click register, you will unable to register with online access to data later. (If you chose this option by accident and would like to start over, please see **PUT LINK ON HOW TO START OVER**).
+
+
+    #. **Register with online access to data**. Choose this option if you're an administrator of larger projects. This option allows you to access your uploaded data and connect multiple installations to the same account. 
+
+How to register your device with online access to data
+########################################################
+
+
+**system currently down, this needs updating later**
+
+
+
+
+Post Registration Setup
+-------------------------------------------
+Now that you have registered successfully, it's time to configure your local KA Lite installation to suit your needs. If any terms like 'facility' or 'device' become confusing, feel free to reference the `Administrator Glossary`_ for a quick reminder.
+
+
+Create a Facility
```

Comment:
This whole page is ripe for screenshots... Here, for example, would be a good place to have one. One thing we can do in anticipation of getting the screenshot directive up and running is to look up which relative url a relevant screenshot would come from and what series of actions we'll need to take to navigate to/build the screenshot.

For example, to get a screenshot of the Create A Facility page, we'll need to start at the relative url `/management/zone/`, click on the link element with the class `create-facility`, and then once we're there we'll need to supply values for the input fields with ids `id_input`, `id_description`, etc. One thing to avoid is any reference to uuids in the database, since we can't guarantee those db rows will exist in the temporary database created for screenshots, so avoid urls like `/securesync/zone/c006e0b178ca570d8474b7c705b1690a/facility/new/`.

I'll share the source for the screenshot directive shortly so we can start to get an idea of how to use it.